db/migrate/20170718142148_create_katello_subscription_facet_pools.rb

```diff
@@ -0,0 +1,15 @@
+class CreateKatelloSubscriptionFacetPools < ActiveRecord::Migration
+  def change
```

Comment:
Is there any cleanup work to do on any old tables? Not too familiar with this part of katello.